crates/pedersen/src/curve_consts_gen.rs

```diff
@@ -0,0 +1,102 @@
+use crate::curve::*;
+use std::fs;
+
+#[allow(dead_code)]
+pub fn generate_consts(path: &str, bits: u32) {
+    let buf = &mut String::with_capacity(10 * 1024 * 1024);
+
+    buf.push_str("use crate::curve::AffinePoint;\n\n");
```

Comment:
```suggestion
    buf.push_str("//! Autogenerated, don't modify.\n");
    buf.push_str("//! See [`crate::curve_consts_gen::tests::generate_curve_constants`].\n");
    buf.push_str("//! The constants are used by [`crate::hash::pedersen_hash_preprocessed`].\n\n");
    buf.push_str("use crate::curve::AffinePoint;\n\n");
```

This should add hints on anyone just stumbling in on the file wondering what it is for.